### PR TITLE
init_project

### PR DESCRIPTION
**Pull request by caovanson06112001**
Screenshot from 2022-08-01 09-02-22

### CODE REVIEW

**Comment by caovanson06112001** [comment]
ready

**Comment by caovanson06112001** [comment]
ready

**Comment by caovanson06112001** [comment]
ready

**Comment by caovanson06112001** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>LibraryBooks</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (ghost) [inline app/views/layouts/application.html.erb]: Dùng "

**Review by ghost — COMMENTED** [review]
cmt

**Comment by caovanson06112001** [comment]
ready

**Pull request closed without merging**